eval/measure_judged.py

```diff
@@ -30,6 +30,8 @@ def load_qrels(path: str) -> Dict[str, Set[str]]:
             qrels[query_id].add(doc_id)
             if i % 1000000 == 0:
                 print('Loading qrels {}'.format(i))
+
+    print('Loading qrels completed')
```

Comment:
Since loading qrels is so fast (even for MS MARCO), may remove all logging here?